.github/workflows/demo_test.yml

```diff
@@ -621,3 +621,22 @@ jobs:
 
     - name: Run unix socket test
       run: docker exec cross_world_unix_socket bash -c "cd /root/occlum/demos/cross_world_uds; SGX_MODE=SIM ./run_cross_world_uds_test.sh"
+
+  Gvisor_syscalls_test:
+    runs-on: ubuntu-18.04
```

Comment:
Please update the version to ubuntu-20.04

Reply:
Modification has been completed